### ISSUE TITLE
Docs for drawBot as a module

### ISSUE THREAD

**Issue by typemytype** [typemytype/drawbot#68]
Little how to install it and use it without the app

**Comment by davidbenque** [comment]
yes please! I've been looking for this and hard to find any info.

**Comment by typemytype** [comment]
see https://github.com/typemytype/drawbot/blob/master/README_MODULE.md

**Comment by justvanrossum** [comment]
That link is dead.

**Comment by typemytype** [comment]
that merged into the main readme: see https://github.com/typemytype/drawbot/blob/master/README.md#compile-drawbot-from-source

but indeed this could be added to the docs
